nfc_ros/README.md

```diff
@@ -0,0 +1,27 @@
+# nfc_ros
+
+This package is for nfc communication.
+
+## How to use
+
+Before use this, please install nfcpy and setup udev for your device.
```

Comment:
why we need to install `nfcpy` manuall, does `catkin_virtualenv` did not work?

Reply:
It does. but we need run `nfcpy` in order to configure udev for NFC device.
I change that section to use the virtualenv generated with catkin_virtualenv.